src/persistence/common.jl

```diff
@@ -34,20 +66,11 @@ save(io::IO, g::Graph, t::Symbol=:lg) = save(io, g, "graph", t)
 save(io::IO, g::DiGraph, t::Symbol=:lg) = save(io, g, "digraph", t)
 
 # save a dictionary of graphs {"name" => graph}
-"""Saves multiple graphs in a dictionary mapping graph name to graph to stream
-`io` using graph type `t`."""
 function save(io::IO, d::Dict{AbstractString, SimpleGraph}, t::Symbol=:lg)
     t in keys(filemap) || error("Please select a supported graph format: one of $(keys(filemap))")
     return filemap[t][4](io, d)
 end
 
-# load from a file
-function load(fn::AbstractString, x...)
-    GZip.open(fn,"r") do io
-        load(io, x...)
-    end
-end
-
 # save to a file
```

Comment:
Ah, I see it. Thanks. But this is the general problem with making changes that aren't reflected in the PR notes. If you're going to do this (move things around), please be clear in the submission so that I can keep an eye out for things that aren't supposed to be changed.